### DISCUSSION TITLE
Tour Component similar to AntD's Tour

### DISCUSSION THREAD

**Discussion by pertinate** [shadcn-ui/ui#3749] [Ideas]
A tour component would be a good addition to the standard shadcn component ecosystem.

I wanted to take a stab at creating the component myself and would like some feedback before moving further.

I'd appreciate any criticisms!

https://github.com/pertinate/ui/blob/feature/tour/apps/www/registry/new-york/ui/tour.tsx
https://github.com/pertinate/ui/blob/feature/tour/apps/www/registry/new-york/example/tour-demo.tsx

I recorded a demo here from my personal project here: https://www.youtube.com/watch?v=ZcwmqjrOZ6w

**Comment by martmalo** [comment]
This is really nice idea!
